registration/models.py

```diff
@@ -305,7 +303,7 @@ def domain(self):
         """
         The full domain requested for this application.
         """
-        return '{0}.{1}'.format(self.subdomain, self.BASE_DOMAIN)
+        return '{0}.{1}'.format(self.subdomain, settings.DEFAULT_INSTANCE_BASE_DOMAIN)
```

Comment:
I don't know why `self.BASE_DOMAIN` was here in addition to being in settings (`DEFAULT_INSTANCE_BASE_DOMAIN`), but I guess it's because of Ocim stage: Ocim stage has the .env setting `DEFAULT_INSTANCE_BASE_DOMAIN=stage.opencraft.hosting`, but the code was hardcoding `opencraft.hosting` (in `self.BASE_DOMAIN`) maybe as a way to make Ocim Stage show the same behaviour as Ocim production, which can be desirable.
With the new code, Ocim Stage users would get `mysubdomain.stage.opencraft.hosting` by default whereas they were getting `mysubdomain.opencraft.hosting` before. I think this difference  doesn't matter because we don't have users registering in Stage, much less with custom domains. But it will make a difference when we test this PR in Stage.

Reply:
Yes, I agree. We will need to make sure testing on stage properly, to not break even our own users experience on stage. 😇

Reply:
By the way, I just checked. The domain names we registered for OCIM stage environments are ending with `stage.opencraft.hosting`. Soooo, this may not be an issue

<img width="358" alt="Screenshot 2020-09-28 at 7 57 45" src="https://user-images.githubusercontent.com/19173947/94395588-4de0d300-0160-11eb-8145-553b67412543.png">